src/ts/components/account/Progress.tsx

```diff
@@ -11,6 +15,17 @@ interface IProgressProps {
  */
 class Progress extends React.Component<IProgressProps> {
 
+  handleClick = route => event => {
+    // only allow to go back
+    if (this.props.progress > 1 && route === CREATE_PASSWORD_ROUTE) {
```

Comment:
so progress 1 => create pwd, 2 => generate phrase, 3 => confirm phrase? for some reason, `confirm phrase` seems not change progress icon.

Reply:
![image](https://user-images.githubusercontent.com/37568337/56502527-edef0180-6555-11e9-9f9a-d047cf8f104d.png)

Reply:
According to the screen shot, confirm screen is still under progress step 2.

Reply:
So the new account created page will be step 3?

Reply:
there is another step (step 3) to select chain according to the screenshot

Reply:
I found this is a bit confusing as well, but then I checked zeplin mockup and found it was like this

Reply:
![image](https://user-images.githubusercontent.com/37568337/56503013-c39e4380-6557-11e9-856b-fcab7cdeae59.png)
yeah. no one actually on step 3 :)

Reply:
<img width="601" alt="Screen Shot 2019-04-22 at 1 43 55 pm" src="https://user-images.githubusercontent.com/20808865/56503163-fd6f4a00-6557-11e9-98d1-5fbcd1b16f85.png">

Reply:
<img width="379" alt="Screen Shot 2019-04-22 at 11 39 59 pm" src="https://user-images.githubusercontent.com/20808865/56503182-12e47400-6558-11e9-9e7d-0cd7e9173cf9.png">

Reply:
i see. I think we need to revisit this later.